artifacts/notification-as-cloud-event.yaml

```diff
@@ -89,23 +89,23 @@ paths:
         201:
           description: Created
           headers:
-            X-Correlator:
-              $ref: "#/components/headers/X-Correlator"
+            x-correlator:
+              $ref: "#/components/headers/x-correlator"
         202:
           description: Accepted
           headers:
-            X-Correlator:
-              $ref: "#/components/headers/X-Correlator"
+            x-correlator:
+              $ref: "#/components/headers/x-correlator"
         204:
           description: No Content
           headers:
-            X-Correlator:
-              $ref: "#/components/headers/X-Correlator"
+            x-correlator:
+              $ref: "#/components/headers/x-correlator"
         200:
           description: Ok
           headers:
-            X-Correlator:
-              $ref: "#/components/headers/X-Correlator"
+            x-correlator:
+              $ref: "#/components/headers/x-correlator"
           content:
             application/json:
               schema:
```

Comment:
Regarding Exceptions alignment with [PR#189](https://github.com/camaraproject/Commonalities/pull/189):
- `400`, `401`, `403`, `500`, `503` are clear
- `415`, `429` think makes sense to have in PR#189 as well
- `410` not sure about UC for 410. In case we keep it, we should align in PR#189 as well

cc @shilpa-padgaonkar, @patrice-conil, @bigludo7, @rartych, @jlurien

Reply:
Ok for me to add 415 & 429 in PR#189 event if I'm bit dubious about the UC. I will do.

For 410 I'm more in favor to not add it but will follow your guidance.

Reply:
My understanding:
415 - would be the case when API Consumer (Server Side in this flow) does not support "application/json" -> This in principle should never happen because is a guideline of CloudEvent CAMARA proifle
429 - would be the case when API Consumer (Server Side in this flow) receiving many requests over its threshold (could happen)

@patrice-conil Do you have context abiut why 410, 415, 429 were included in the initial proposal? https://github.com/camaraproject/Commonalities/pull/43/commits/339a8e2410dac6be4f6e533673ca03a27447edd4

Reply:
410 - The unique case I think it may apply is the case of that callback no longer available. I mean the Server side decided to switch-off that endpoint. This would be differnet to a 503, because in 503 endpoint exists but is not available temporarily (regular exception for maintenance tasks)

Reply:
So following your explanation @PedroDiez  I understand that we should have 429 & 410 and drop 415.

Reply:
Hi @PedroDiez,
My understanding:
410 was introduced to handle cases where the callback URL is no longer available.
415 is what the server side should respond if Content-type or Accept are not "application/json" (should never happen) .
429 could be used to handle congestion on the callback endpoint.

Reply:
Hi @patrice-conil, thanks for the clarification! I think we need to align with #189 (POST callback) cc @bigludo7 

My view:
- Keep 410 for the case the endpoint is no longer available (Not 503 case because that one is temporal). Benefit is to have an indication for Telco Operation teams to enable operations procedure to the Subscriber to say "Please be aware there is a flow no longer working, you will need to generate a new subscription with a new callback or in the case of implicit flow to indicate new callback"

- Would take out 415 - Should never happen because developers MUST adopt CAMARA cloudevent profile (if you think/prefer to have for safety/operation reasons, i.e. to advise an App developer that need to support application/json as a warning to adapt its side to be able to interoperate it is fine)

- Keep 429

WDYT?

Reply:
Done